2022-02-23 19:04:30,511 INFO    MainThread:9467 [wandb_setup.py:_flush():71] setting env: {}
2022-02-23 19:04:30,511 INFO    MainThread:9467 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-23 19:04:30,511 INFO    MainThread:9467 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220223_190430-24zfl56t/logs/debug.log
2022-02-23 19:04:30,512 INFO    MainThread:9467 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220223_190430-24zfl56t/logs/debug-internal.log
2022-02-23 19:04:30,512 INFO    MainThread:9467 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f90142b9650>
2022-02-23 19:04:30,512 INFO    MainThread:9467 [wandb_init.py:init():404] calling init triggers
2022-02-23 19:04:30,513 INFO    MainThread:9467 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-23 19:04:30,513 INFO    MainThread:9467 [wandb_init.py:init():460] starting backend
2022-02-23 19:04:30,513 INFO    MainThread:9467 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 19:04:30,590 INFO    MainThread:9467 [backend.py:ensure_launched():216] starting backend process...
2022-02-23 19:04:30,636 INFO    MainThread:9467 [backend.py:ensure_launched():222] started backend process with pid: 9512
2022-02-23 19:04:30,638 INFO    MainThread:9467 [wandb_init.py:init():469] backend started and connected
2022-02-23 19:04:30,673 INFO    MainThread:9467 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-23 19:04:30,674 INFO    MainThread:9467 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-23 19:04:30,674 INFO    MainThread:9467 [wandb_init.py:init():533] updated telemetry
2022-02-23 19:04:30,771 INFO    MainThread:9467 [wandb_init.py:init():563] communicating current version
2022-02-23 19:04:32,554 INFO    MainThread:9467 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-23 19:04:32,556 INFO    MainThread:9467 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-23 19:04:32,923 INFO    MainThread:9467 [wandb_init.py:init():606] starting run threads in backend
2022-02-23 19:04:37,944 INFO    MainThread:9467 [wandb_run.py:_console_start():1810] atexit reg
2022-02-23 19:04:37,945 INFO    MainThread:9467 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-23 19:04:37,946 INFO    MainThread:9467 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-23 19:04:37,947 INFO    MainThread:9467 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-23 19:04:37,947 INFO    MainThread:9467 [wandb_init.py:init():633] run started, returning control to user process
2022-02-23 19:04:38,232 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:04:38,232 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:05:22,521 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:05:22,524 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:05:22,524 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:05:52,831 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:05:52,854 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:05:52,854 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:05:57,004 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:05:57,035 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:05:57,035 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:06:03,850 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:06:03,870 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:06:03,871 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:06:08,504 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:06:08,506 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:06:08,507 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:06:11,118 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:06:11,127 INFO    MainThread:9467 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 19:06:11,127 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:06:11,216 INFO    MainThread:9467 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-23 19:06:11,239 INFO    MainThread:9467 [wandb_setup.py:_flush():71] setting env: {}
2022-02-23 19:06:11,239 INFO    MainThread:9467 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-23 19:06:11,240 INFO    MainThread:9467 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220223_190611-11uvw91d/logs/debug.log
2022-02-23 19:06:11,240 INFO    MainThread:9467 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220223_190611-11uvw91d/logs/debug-internal.log
2022-02-23 19:06:11,240 INFO    MainThread:9467 [wandb_init.py:init():404] calling init triggers
2022-02-23 19:06:11,240 INFO    MainThread:9467 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-23 19:06:11,240 INFO    MainThread:9467 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 24zfl56t
2022-02-23 19:06:11,243 INFO    MainThread:9467 [wandb_run.py:finish():1457] finishing run simon-donike/test/24zfl56t
2022-02-23 19:06:11,244 INFO    MainThread:9467 [jupyter.py:save_history():429] not saving jupyter history
2022-02-23 19:06:11,244 INFO    MainThread:9467 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 19:06:11,244 INFO    MainThread:9467 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-23 19:06:11,244 INFO    MainThread:9467 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-23 19:06:11,246 INFO    MainThread:9467 [wandb_run.py:_restore():1752] restore
2022-02-23 19:06:12,050 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-23 19:06:14,917 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-23 19:06:15,032 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-23 19:06:15,138 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-23 19:06:15,242 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-23 19:06:15,345 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:15,449 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:15,551 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:15,654 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:15,757 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:15,860 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:16,108 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-23 19:06:16,606 INFO    MainThread:9467 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}
local_info {
}

2022-02-23 19:06:18,638 INFO    MainThread:9467 [wandb_run.py:_append_files():2180] logging synced files
